modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -4,73 +4,68 @@ This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://
 
 == Prerequisites
 
-Before provisioning an FCOS machine, you must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+Before provisioning an FCOS machine, it is recommended to have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
 
 If you do not want to use Ignition to get started, you can make use of the https://coreos.github.io/afterburn/platforms/[Afterburn support] and provide an SSH key via the cloud provider and continue from there.
 
 You also need to have access to an Exoscale account. https://portal.exoscale.com/register[Register] if you don't have one.
 
-== About images
```

Comment:
I'm on the fence on that. I tend to think it's better to work out the differences with the cloud provider in a less user facing way.

I think it's pretty clear to the user without us having to mention here because they mention `34` and `33` in the name.

Reply:
Well, I don't think we are blaming anyone. Just avoid future questions / issues. Maybe we could write "currently out of date, issue in progress in #..."?

Reply:
Added a note back in and a link to https://github.com/coreos/fedora-coreos-tracker/issues/1166